detector/wordpress.go

```diff
@@ -191,23 +211,65 @@ func extractToVulnInfos(pkgName string, cves []WpCveInfo) (vinfos []models.VulnI
 		}
 
 		var refs []models.Reference
+		if vulnerability.Poc != nil {
+			refs = append(refs, models.Reference{
+				Source: *vulnerability.Poc,
+			})
+		}
 		for _, url := range vulnerability.References.URL {
 			refs = append(refs, models.Reference{
 				Link: url,
 			})
 		}
+		for _, id := range vulnerability.References.YouTube {
+			refs = append(refs, models.Reference{
+				Link: fmt.Sprintf("https://www.youtube.com/watch?v=%s", id),
+			})
+		}
+		for _, id := range vulnerability.References.ExploitDB {
+			refs = append(refs, models.Reference{
+				Link: fmt.Sprintf("https://www.exploit-db.com/exploits/%s", id),
+			})
+		}
```

Comment:
I think exploit information should be stored here.
https://github.com/future-architect/vuls/blob/7d5a47bc3354da4d8cfbef2200c916663d72a2b4/models/vulninfos.go#L264

Reply:
I overlooked the Exploit struct was prepared.
Thank you for pointing it out. I will correct the PR.